worker/pkg/benthos/transformers/gen_generate_bool.go

```diff
@@ -39,7 +39,7 @@ func NewGenerateBoolOpts(
 func (t *GenerateBool) GetJsTemplateData() (*TemplateData, error) {
 	return &TemplateData{
 		Name: "generateBool",
-		Description: "Generates a boolean value at random.",
+		Description: "Generates a random boolean value.",
```

Comment:
Hm, where are the source changes that caused all of the generated code to update? 
Or was it made in a past PR and we don't have the generation fully automated yet?
@alishakawaguchi

Reply:
These changes must have gotten picked up by accident - only meant to change the empty states. But in any case, the re-wording is valid. It aligns our docs with our descriptions in the benthos spec.